### ISSUE TITLE
Issue with Power BI HierarchySlicer Search Feature

### ISSUE THREAD

**Issue by fdgsogc91** [liprec/azurebi-docs#127]
When using the Search feature in the Power BI HierarchySlicer, it can only search one word at a time.  The expectation is that as I type, say the name of a customer, where I am using the customer name field in the filter, that it would narrow the search down the more words I type.  However, the search feature stops searching as soon as you enter a second word.  This is not expected and is driving customers crazy. See example below.

![image](https://user-images.githubusercontent.com/32048899/90793075-457fb780-e2d9-11ea-8cc7-8d61cd2234e4.png)

---
#### Document details

⚠ Do not edit this section. It is required for docs.jppp.org ➟ GitHub issue linking.

* document_id=487cd175-be13-40df-a80e-aa1c7f531eb1
* document_url=https://azurebi-docs.jppp.org/powerbi-visuals/hierarchy-slicer.html

**Comment by liprec** [comment]
The current version (v2.2.1) does have an update to the searching options and should also fix this issue.
Can you please validate?

Thanks, JP

**Comment by bravespiano5** [comment]
Awesome this has been resolved.

I still only see v2.2.2.0 available in app source however still?
![image](https://user-images.githubusercontent.com/55611054/98280773-e2153580-1f69-11eb-85a0-86d53f25d6a7.png)

**Comment by liprec** [comment]
Yes, **v2.2.2** is the latest and upcoming version. The additional search features are already in the current **v2.2.1** version.
